### PR TITLE
split out api service to its own repository

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: ansible/roles/website/tasks/main.yml [hunk 1]
```diff
@@ -29,10 +29,25 @@
...
+    file:
+      path: '{{ api_dir }}'
+      owner: root
+      group: root
+      state: directory
+
+  - name: Clone Git repository of the API
+    git:
+      dest: '{{ api_dir }}'
+      repo: https://github.com/mmoll/openrailwaymap-node-api.git
```
Comment (mmoll) [inline ansible/roles/website/tasks/main.yml:42]: Needs obviously to be created in the OpenRailwayMap organization.

**Comment by mmoll** [comment]
Question: is `/var/log/orm-api` really a directory on the server? The rsyslogd configuration would create it as a file, so at least a new install ends up with a directory, but rsyslogd putting nothing into it.

**Comment by mmoll** [comment]
@Nakaner @DerDakon Any comment on this? :)

**Comment by DerDakon** [comment]
It's an empty directory.

**Comment by mmoll** [comment]
> It's an empty directory.

OK, so the question would be: Should it be a file, so the node service can log into it or should the (currently broken) logging to that location be removed?

**Pull request closed without merging**